contracts/src/Proofs.sol

```diff
@@ -99,6 +99,12 @@ library Hashes {
             }
             
             value := mload(0x00)
+            // SHA254 hash for compatibility with Filecoin piece commitments.
+            // "The Sha254 functions are identical to Sha256 except that the last two bits of the Sha256 256-bit digest are zeroed out."
+            // The bytes of uint256 are arranged in big-endian order, MSB first in memory.
+            // The bits in each byte are arranged in little-endian order.
+            // Thus, the "last two bits" are the first two bits of the last byte.
+            value := and(value, 0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF3F)
```

Comment:
This is fine but maybe consider extracting to a constant and moving comment along with it.  Im thinking this because we use this constant again in test